mysql-test/t/bug91091.test

```diff
@@ -0,0 +1,60 @@
+--source include/have_innodb.inc
```

Comment:
Please move to innodb suite

Reply:
Initially I thought that this problem was collation-related and I put the test case to the `main` suite. The problem turned out to be in InnoDB code. Moving to the `innodb` test suite.